UI/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="static/css/index.css">
+    <title>Document</title>
+</head>
+<body>
+        <nav>
+                <div class="logo"><a href="#"><img src="static/images/logo.png"></a>
+                   </div>
+             
+                   <h1>Store Manager</h1>
+                   <div class="btn">
+                       <a href="login.html" class="btn">
+                           Signin
```

Comment:
Signin should be two words, i.e. _Sign In_

Reply:
Thanks for this